docs/src/snap/explanation/epa.md

```diff
@@ -0,0 +1,548 @@
+# Enhanced Platform Awareness
+
+Enhanced Platform Awareness (EPA) represents a methodology and a set of
+enhancements across various layers of the orchestration stack, aimed at
+optimising platform capability, configuration, and capacity usage. For
+telecommunications service providers using virtual network functions (VNFs) to
+deliver services, EPA offers enhanced application performance, increased
+input/output throughput, and improved determinism.
```

Comment:
It's the primary case that we know of for our Kubernetes, but not necessarily the only one. Deterministic models are used in control systems and other areas where it may be useful. I will take the point and reword slightly though